### ISSUE TITLE
lint script

### ISSUE THREAD

**Issue by mattlub** [FACK1/FoodRecipes#38]
https://github.com/FACK1/FoodRecipes/blob/4d40673dea74a0d60dcc27d0329947d29312395c/package.json#L8

why does your lint script only look inside `src/`?

**Comment by mhmdtshref** [comment]
That's because we've used the ES6 linter test, but as i know we shouldn't use ES6 on the front-end files, we should just use ES5 and that makes an linter errors.

**Comment by mattlub** [comment]
yes, that's ok, but you should also lint the other folders in your `src/` folder!
